### PR TITLE
Add how to revert purple blog post

### PR DESCRIPTION
**Pull request by Samq64**
Graham suggested this on Discord.

### CODE REVIEW

File: content/blog/revert-purple.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+1. Open the settings page and search for "dark mode".
+2. Enable the "Editor dark mode and customizable colors" addon.
+3. Select the "Scratch's default colours (blue)" preset near the bottom of the addon's card.
+4. Do the same for the "Website dark mode and customizable colors" addon.
+5. To make the older pages such as profiles or the discussion forums more consistent with newer ones, enable the "Scratch 2.0 → 3.0 addon".
+
+Now you make projects and browse Scratch with the colours you're used to.
+
+We may also add an addon that removes the text or icons from the editor menu bar to make it more compact and for those who don't like the icons.
```
Comment (CST1229) [inline content/blog/revert-purple.md:17]:
```suggestion
We may also add an addon in the future that removes the text or icons from the editor menu bar to make it more compact and for those who don't like the icons.
```

File: content/blog/revert-purple.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+title: How to revert Scratch's purple changes
+description: Do you dislike Scratch's new purple color used throughout the website and editor? If so, Scratch Addons makes it easy to switch back to blue, or use any other color!
+date: 2023-06-28T00:00:00Z
+author: Samq64
+---
+Do you dislike Scratch's new purple color used throughout the website and editor? If so, Scratch Addons makes it easy to switch back to blue, or use any other color!
+
+1. Open the settings page and search for "dark mode".
+2. Enable the "Editor dark mode and customizable colors" addon.
+3. Select the "Scratch's default colours (blue)" preset near the bottom of the addon's card.
```
Comment (mxmou) [inline content/blog/revert-purple.md:11]: Could this step also mention the dark mode presets with the blue color?

File: content/blog/revert-purple.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+author: Samq64
+---
+Do you dislike Scratch's new purple color used throughout the website and editor? If so, Scratch Addons makes it easy to switch back to blue, or use any other color!
+
+1. Enable the "[Editor dark mode and customizable colors](https://scratch.mit.edu/scratch-addons-extension/settings#addon-editor-dark-mode)" addon.
+2. Select the "Scratch's default colours (blue)" preset. There are also blue dark mode presets available if you prefer.
+3. Do the same for the "[Website dark mode and customizable colors](https://scratch.mit.edu/scratch-addons-extension/settings#addon-dark-www)" addon.
+4. To make the older pages such as profiles or the discussion forums more consistent with newer ones, enable the "[Scratch 2.0 → 3.0 addon](https://scratch.mit.edu/scratch-addons-extension/settings#addon-scratchr2)".
+
+Now you make projects and browse Scratch with the colours you're used to.
```
Comment (GrahamSH-LLK) [inline content/blog/revert-purple.md:14]:
```suggestion
Now you can make projects and browse Scratch with the colours you're used to.
```

**Comment by Hans5958** [comment]
I would like to have a mention of the SA's update, something like "we have updated SA in response to this" r something light.

Also, just a picky suggestion, but can we have a cover image?

**Pull request merged**